docs/docusaurus.config.js

```diff
@@ -90,12 +89,6 @@ module.exports = {
           position: 'left',
           href: `${SWAP_URL}/docs/rasa-enterprise/`,
         },
-        {
-          target: '_self',
-          label: 'Rasa Action Server',
-          position: 'left',
-          href: 'https://rasa.com/docs/action-server',
-        },
```

Comment:
I am not familiar with docusaurus, what does this removal do? Remove the additional top tab on the page?